sdk/attestation/azure-security-attestation/test/samples/attestation/attestation_collateral.hpp

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+#pragma once
+
+#include <azure/core/base64.hpp>
+
+namespace Azure { namespace Security { namespace Attestation { namespace Samples {
+
+  class AttestationCollateral {
```

Comment:
nit.
Add docs.
It is not mandatory for Tests or Samples, but I noticed you did a great job on the samples source...  so, might be worth it to complete the work here ( but up to you)